docs/profiles.md

```diff
@@ -31,12 +31,12 @@ Attribute                     | Type     | Notes
 ------------------------------|----------|----------------------------------------------------------------------------
 weight_name                   | String   | Name used in output for the routing weight property (default 'duration')
 weight_precision              | Unsigned | Decimal precision of edge weights (default 1)
-left_hand_driving             | Boolean  | Are vehicles assumed to drive on the left? (used in guidance)
-use_turn_restrictions         | Boolean  | Are turn instructions followed?
-continue_straight_at_waypoint | Boolean  | Must the route continue straight on at a via point, or are U-turns allowed?
+left_hand_driving             | Boolean  | Are vehicles assumed to drive on the left? (used in guidance, default false)
```

Comment:
@TheMarex Do you mean boolean values specifically or any?

I followed the status quo, namely these are not code-tagged
* weight_name: default 'duration'
* weight_precision: default 1